service/settings/base.py

```diff
@@ -155,7 +155,11 @@
 
 AUTH_USER_MODEL = 'api.User'
 
-SITE_ID = 1
+FIXTURE_DIRS = (
+   '/api/fixtures/',
+)
+
+SITE_ID = 2
```

Comment:
This should probably go back to `SITE_ID = 1`, otherwise I could see this breaking someone else's config